src/android/toga_android/window.py

```diff
@@ -26,7 +26,9 @@ def set_app(self, app):
         self.app = app
 
     def set_content(self, widget):
-        self.app.native.setContentView(widget.native)
+        self.widget = widget
```

Comment:
For naming consistency with other platforms, this should be `self.content`, if it's retained at all (IIRC most platforms just retain whatever "native" representation of content exists.